<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>planRegister</class>
 <widget class="QDialog" name="planRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plan Register</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>23</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>52</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>85</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Place</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDate">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>21</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>2018-01-01</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTime">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>52</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>7 am-10 am</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePlace">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>84</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbUpdate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>116</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>116</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lwPlan">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>21</y>
     <width>91</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAdd">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>116</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
